docker/Dockerfile-epoch

```diff
@@ -21,12 +21,17 @@ RUN useradd --shell /bin/bash epoch \
 USER epoch
 ENV SHELL /bin/bash
 
+# Create data directories in advance so that volumes can be mounted in there
+# see https://github.com/moby/moby/issues/2259 for more about this nasty hack
+RUN mkdir -p /home/epoch/node/data/mnesia \
+    && mkdir -p /home/epoch/node/keys
```

Comment:
I noticed that if I run `docker-compose up -d` without `docker-compose build` then `curl http://localhost:3002/internal/v2/account/pub-key` does not work. The reason is that nodes 1 2 3 die straight away not being able to write in DB dir ("Not allowed to write in Mnesia dir"). Though `docker-compose build` includes this change in the Docker image that enables it to work. Ok.

Reply:
Indeed you need the latests image of build by this Dockerfile.

Reply:
Little suggestion here: Maybe we can mark the directories as volumes? As these directories are important, they should not be deleted by docker without the users consent (docker rm -v), right?